Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -469,7 +429,8 @@ class CombinatorialKalmanFilter {
                        << result.lastTrackIndices.size() << " tracks");
           result.filtered = true;
         } else {
-          // remove the active tip and continue with the next
+          storeLastActiveTip(result);
```

Comment:
Why do we need to store here in the new version?

Reply:
This is one of the cases where we end the propagation of the current track hypothesis and jump to another one. Not storing a potential track here can cause inefficiency and throws away all the computation we did before.